app/views/admin/facilities/index.html.erb

```diff
@@ -23,7 +23,7 @@
     </div>
 
     <p>
-        Protocol: <%= facility_group.protocol.name %>
+        Protocol: <%= facility_group.protocol&.name %>
```

Comment:
`&.` is unnecessary. 

We should also avoid using `&.` and instead handle the nil case as a condition.

Reply:
Removed the `nil` punning since it is logically impossible for the protocol to be `nil`.